pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelTest.java

```diff
@@ -553,11 +552,24 @@ public void splitAndRetryTest() throws Exception {
             }
             // Call the real method
             reset(namespaceService);
+            doReturn(CompletableFuture.completedFuture(List.of("test-topic-1", "test-topic-2")))
+                    .when(namespaceService).getOwnedTopicListForNamespaceBundle(any());
             return future;
         }).when(namespaceService).updateNamespaceBundles(any(), any());
         doReturn(namespaceService).when(pulsar1).getNamespaceService();
-
-        Split split = new Split(bundle, ownerAddr1.get(), new HashMap<>());
```

Comment:
Like commented above, the split scheduler passes an empty map, and this BSC test case validates if the split event completes with an empty map.

Please provide another UT for the non-empty case(admin split case).

Reply:
Updated.